README.md

```diff
@@ -147,4 +147,63 @@ for node in folderContents:
 
 # download content of node into provided file
 with open('path/to/' + anyNode.name, 'wb') as f:
-    anyNode.downloadContent(f)
\ No newline at end of file
+    anyNode.downloadContent(f)
+
+
+
+# create a new node into folder and upload data
+from alfpyclient.common.errors import Conflict
+from alfpyclient.common.connections import connect
+...
+
+USER = "username"
+PASSWORD = "password"
+
+client = connect('<serverAddress>/alfresco', USER, PASSWORD)
```

Comment:
This use case is possible, true. Though my goal would be to encapsulate it in an operation on the parent folder for 90+% of use cases, instead of requiring the library user to call a low-level multipart post.

Reply:
Ok, the goal was just to explain my new function... Let you add explanations about this. Should i remove my text in the README file ?

Reply:
No need to remove the text from the README. It was just meant as an FYI - that's why I did not mention this in the overall "request changes" review comment.